db/migrate/20230614144145_create_account_config.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAccountConfig < ActiveRecord::Migration[6.1]
+  def change
+    create_table :account_configs do |t|
+      t.string :account_id, null: false, index: { unique: true }
+      t.text :description, null: false
+      t.string :scope, array: true, null: false
+      t.string :access_token_audience, null: false
+      t.string :certificates, array: true, null: false
```

Comment:
Do we want this to be `null: false`? Scenario: we set up a new AccountConfig for someone then they would come in later and add their certs

Reply:
Yeah that makes sense, there's a method that converts the string keys into OpenSSL objects so if nothing's there it can be handled there.